grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -334,36 +375,114 @@ def _import_file(self, filename, module, args):
+                                           output=shadows_imported,
+                                           **self._args)
+
+                        gs.mapcalc(f'{shadows_selected} = if({shadows_imported} == 3, 2, 0)')
+                        gs.mapcalc(f'{mask_selected} = max({shadows_selected},{clouds_selected})')
+                    except:
+                        gs.warning(_(f'Unable to import shadows for {"_".join([items[5], items[2]])}'))
+
+                else:
+                    gs.run_command("g.rename", quiet=True,
+                                   raster=(clouds_selected, mask_selected))
+
+                # Cleaning small patches
+                gs.run_command('r.reclass.area',
+                               input=mask_selected,
+                               output=mask_cleaned,
+                               value=area_threshold,
+                               mode='greater')
+
+                # Extract clouds (and shadows) as mask
+                reclass_lookup = ['2 = 2 shadows', '1 = 1 clouds', '0 = NULL']
+
+                mask_extract = gs.feed_command('r.reclass', input=mask_cleaned, output=mask_selected, rules="-", overwrite=True)
+                mask_extract.stdin.write("\n".join(reclass_lookup).encode())
+                mask_extract.stdin.close()
+                mask_extract.wait()
+
+                info_stats = gs.parse_command('r.stats', input=mask_selected, flags='p')
+
+                # Create final cloud (and shadow) mask & display areal statistics
+                if output == 'vector':
+                    gs.run_command('r.to.vect', input=mask_selected, output=map_name, type='area', flags='s')
+                    gs.run_command('v.db.addcolumn', map=map_name, columns='GRASSRGB varchar(20)', quiet=True)
+                    gs.run_command('v.db.update', map=map_name, column='GRASSRGB', where='label=="clouds"', value='230:230:230', quiet=True)
+                    gs.run_command('v.db.update', map=map_name, column='GRASSRGB', where='label=="shadows"', value='60:60:60', quiet=True)
+                    gs.run_command('v.colors', map=map_name, use='attr', column='value', rgb_column='GRASSRGB', flags='c', quiet=True)
+                    gs.vector_history(map_name)
+
+                else:
+                    gs.mapcalc(f'{map_name} = {mask_selected}')
+                    gs.raster_history(map_name)
+
+                gs.message(_(f'Areal proportion of masked clouds:{[key.split()[1] for key in info_stats][0]}'))
+                if shadows == 'yes':
+                    if len(info_stats) > 2:
+                        gs.message(_(f'Areal proportion of masked shadows:{[key.split()[1] for key in info_stats][1]}'))
+                    else:
+                        gs.message(_('Areal proportion of masked shadows:0%'))
+
+                gs.del_temp_region()
+
+            except:
+                gs.warning(_(f'Unable to import cloud mask for {"_".join([items[5], items[2]])}'))
```

Comment:
It would be helpful to also have a specific warning for the case that someone tries to import a cloud mask from a Level-1C dataset. The level information L2A/L1C can be taken from the scene name